docs/.vuepress/components/DoDont.vue

```diff
@@ -35,32 +35,11 @@ export default {
       type: Array,
     }
   },
-  beforeMount() {
-    this.examples.forEach((example, idx) => {
-      this.$options.components[`comp-${example.type}${idx}`] = { ...Vue.compile(`<span>${example.caption}</span>`) };
```

Comment:
this code is not only pointless, it would break if any examples were removed from the array 😆